labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise02.scala

```diff
@@ -19,7 +19,12 @@ object ListManipulationExercise02 {
    * of the two list
    */
   def sumOfTwo(l1: List[Int], l2: List[Int]): List[Int] = {
-    error("fix me")
+    if (l1.length == 0)
+      return l2
+    if (l2.length == 0)
+      return l1
+    else
+      return (l1.zip(l2)) map ({ case (i:Int, j:Int) => i+j})
```

Comment:
I'm not sure if this is correct on all inputs. What about `[1, 2, 3]` and `[1]`?